src/parsing.jl

```diff
@@ -298,12 +298,14 @@ function pqparse(::Type{Time}, str::AbstractString)
     try
         return parse(Time, str)
     catch err
-        if !(err isa InexactError)
-            rethrow(err)
+        try
+            return parse(Time, _trunc_seconds(str))
```

Comment:
I'm not a fan of how errors from `_trunc_seconds` are effectively ignored by this

Reply:
Would it be worth while to just always call `_trunc_seconds` on `str`?

IE, to change this function to 

```julia
pqparse(::Type{Time}, str::AbstractString) = parse(Time, _trunc_seconds(str))
```